### ISSUE TITLE
NegotiateStream on UWP (WCF Windows authentication over NetTcp)

### ISSUE THREAD

**Issue by cmatj** [dotnet/corefx#14560]
Trying to communicate with a WCF service from a UWP app using `NetTcpBinding` and Windows authentication will throw a `System.PlatformNotSupportedException`. The binding is defined like this:

```c#
   NetTcpBinding binding = new NetTcpBinding();
   binding.Security.Mode = SecurityMode.Transport;
   binding.Security.Transport.ClientCredentialType = TcpClientCredentialType.Windows;
```

The exception will be thrown when trying to invoke any method on the endpoint's interface:
```c#
   var ep = new EndpointAddress("net.tcp://myhost:8733/aaa/MyService");
   var factory = new ChannelFactory<IMyServicee>(binding, ep);
   var service = factory.CreateChannel();

   var result = service.AnyMethod();       // <-- this will throw
```
Thanks
/Thomas

**Comment by CIPop** [comment]
Moved to WCF: https://github.com/dotnet/wcf/issues/1818

I believe that WCF requires `NegotiateStream` (from System.Net.Security) for this to work. NegotiateStream is not currently available on the UWP platform.

**Comment by CIPop** [comment]
My bad: didn't read the title carefully. This is actually to track NegotiateStream not WCF.

**Comment by CIPop** [comment]
Duplicate of #20033. 
After #21283 most of the functionality is ready. One behavior difference was observed within the single-machine tests (multi-machine tests need to be ran as well): #21282
